python/cugraph/cugraph/traversal/ms_bfs.py

```diff
@@ -56,7 +56,6 @@ def _get_feasibility(G, sources, components=None, depth_limit=None):
 
     # Fixme not implemented in RMM yet
```

Comment:
This should probably use `rmm.mr.available_device_memory()` but this function is currently unreachable due to the `NotImplementedError` thrown on line 149 below.